sdk/storage/Azure.Storage.Blobs.Cryptography/src/Models/EncryptionAlgorithm.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Storage.Blobs.Specialized.Cryptography.Models
+{
+    /// <summary>
+    /// Specifies the encryption algorithm used to encrypt a resource.
+    /// </summary>
+    public enum ClientsideEncryptionAlgorithm
```

Comment:
Why isn't this just `EncryptionAlgorithm`?

Reply:
EncryptionAlgorithm is an overloaded term when CPK also exists in the main library. EncryptionAlgorithm is an already existing enum defined by swagger and I wanted to ensure the two aren't confused.

Reply:
It'd be nice of all the encryption names matched up in intellisense so let's try to get this aligned to whatever we call `ClientsideEncryptionOptions`.

Reply:
Also, should this be an extensible enum?